src/base/tests/test_util.py

```diff
@@ -1572,15 +1572,15 @@ def test_m2m_no_conflict(self):
         mapping = {g1.id: g3.id, g2.id: g3.id}
 
         u1 = self.env["res.users"].create({"login": "U1", "name": "U1"})
-        u1.groups_id = g1 | g3
-        self.assertEqual(u1.groups_id.ids, [g1.id, g3.id])
+        u1.group_ids = g1 | g3
```

Comment:
Should still be compatible will old versions.
```suggestion
        groups = "groups_ids" if version_gte("saas~18.1") else "groups_id"
        u1[groups] = g1 | g3
```